src/decoder/plugins/FluidsynthDecoderPlugin.cxx

```diff
@@ -83,6 +84,14 @@ fluidsynth_init(const ConfigBlock &block)
 	soundfont_path = block.GetBlockValue("soundfont",
 					     "/usr/share/sounds/sf2/FluidR3_GM.sf2");
 
+	char *endptr;
+	const char *svalue = block.GetBlockValue("gain", 0.2);
```

Comment:
Where does the `0.2` come from?
I'd prefer not to call `fluid_settings_setnum()` if the user did not specify the setting, and leave the default to the library.

Reply:
@MaxKellermann That is the default value from `FluidSynth` today https://www.fluidsynth.org/api/settings_synth.html#settings_synth_gain

This *should* have zero impact on users if the value isn't set

Reply:
If a new library version changes the default, MPD should use the new default, and not stick to some old default value, so please don't hard-code those defaults in MPD.

Reply:
Fixed! Pulls and checks for `nullptr`